pkg/reconciler/ingress/ingress.go

```diff
@@ -578,3 +583,19 @@ func getLBStatus(gatewayServiceURL string) []v1alpha1.LoadBalancerIngressStatus
 func enablesAutoTLS(ctx context.Context) bool {
 	return config.FromContext(ctx).Network.AutoTLS
 }
+func isIstioInjectionEnabledForNs(r *BaseIngressReconciler, ra ReconcilerAccessor, ia v1alpha1.IngressAccessor) (bool, error) {
+	ing, err := ra.GetIngress(ia.GetNamespace(), ia.GetName())
+	if err != nil {
+		return false, err
+	}
+	ns := ing.GetLabels()[serving.RouteNamespaceLabelKey]
+	realNs, err := r.KubeClientSet.CoreV1().Namespaces().Get(ns, metav1.GetOptions{})
```

Comment:
We probably should use `NamespaceLister`.

Reply:
Thanks for review. At first I wanted to use NamespaceLister. But it seemed that in knative serving, only at this point namespace lister is needed.
I'm not sure whether it need to add such logic for this fix.
If so, I could use NamespaceLister here.